tools/coqdep/common.ml

```diff
@@ -526,21 +528,35 @@ let rec find_dependencies basename =
 
 let write_vos = ref false
 
-let coq_dependencies () =
-  List.iter
-    (fun (name,_) ->
-       let ename = escape name in
-       let glob = if !option_noglob then "" else ename^".glob " in
-       let deps = find_dependencies name in
-       printf "%s.vo %s%s.v.beautified %s.required_vo: %s.v %s\n" ename glob ename ename ename
-        (string_of_dependency_list ".vo" deps);
-       printf "%s.vio: %s.v %s\n" ename ename
-         (string_of_dependency_list ".vio" deps);
-       if !write_vos then
-         printf "%s.vos %s.vok %s.required_vos: %s.v %s\n" ename ename ename ename
-           (string_of_dependency_list ".vos" deps);
-       printf "%!")
-    (List.rev !vAccu)
+module Dep_info = struct
+
+  (* This should be put into a Coq_module.t *)
```

Comment:
?

Reply:
It means that at some point we should have `Coq_module.t` and `name : t \to string` etc...

What do you want me to do with this comment? [I put it as we were doing a hacking session]

Reply:
Is that different from renaming Dep_info to Coq_module?

Reply:
Shouldn't this comment be removed now that there is a `Dep_info.t`?

Reply:
What it means is that instead of `name : string` we should have `module : Coq_module.t` 

`Dep_info.t != Coq_module.t`

Reply:
I can take of it in the depending PR if you want, as you prefer.

Reply:
The comment is misleading then
```suggestion
  (* [name]] should be a [Coq_module.t] (which needs to be defined) *)
```

Reply:
ok fixed.